pkg/events/postprocessing.go

```diff
@@ -153,6 +155,22 @@ func (PostprocessingFinished) Unmarshal(v []byte) (interface{}, error) {
 	return e, err
 }
 
+// PostprocessingRetry is emitted by *some* service which can decide that
+type PostprocessingRetry struct {
```

Comment:
Why not extend (and maybe rename) already existing `ResumePostprocessing` event?

Reply:
@kobergj  `PostprocessingRetry` is an informational event, emitted so that other services can learn about retries happening in the system. It is not used for control flow.
Instead the postprocessing service will, when it detects a `retry` outcome, emit the `PostprocessingRetry` right away and then emit a `StartPostprocessingStep` event after the according backoff period.

Does that make sense to you? If yes I'll try to adjust the event documentation to make that more clear.

Reply:
Ah I see. Who is listening to this event at the moment?

Reply:
It's currently not being received by anyone :)